ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_histograms.py

```diff
@@ -0,0 +1,43 @@
+from turtle import st
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.test_functional.test_core.test_statistical import (
+    statistical_dtype_values,
+)
+import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+#bincount
+
+@handle_frontend_test(
+    fn_tree="numpy.bincount",
+
+    dtype_x_axis = statistical_dtype_values(function="bincount"),
+    dtype = helpers.get_dtypes("integer", full=False, none=True),
+    where = np_frontend_helpers.where(),
+    keep_dims = st.booleans(),
```

Comment:
`@handle_frontend_test` should generate and pass the function's arguments. So, you don't need `dtype`, `where` and `keep_dims`, but you need to add `weights` and `minlength`.